language-plutus-core/src/Language/PlutusCore/Parser.y

```diff
@@ -158,7 +162,17 @@ parseST str = liftErr (runAlexST str (runExceptT parsePlutusCore))
 -- >>> parse "(program 0.1.0 [(con addInteger) x y])"
 -- Right (Program (AlexPn 1 1 2) (Version (AlexPn 9 1 10) 0 1 0) (Apply (AlexPn 15 1 16) (Apply (AlexPn 15 1 16) (Constant (AlexPn 17 1 18) (BuiltinName (AlexPn 21 1 22) AddInteger)) (Var (AlexPn 33 1 34) (Name {nameAttribute = AlexPn 33 1 34, nameString = "x", nameUnique = Unique {unUnique = 0}}))) (Var (AlexPn 35 1 36) (Name {nameAttribute = AlexPn 35 1 36, nameString = "y", nameUnique = Unique {unUnique = 1}}))))
 parse :: BSL.ByteString -> Either ParseError (Program TyName Name AlexPosn)
-parse str = liftErr (runAlex str (runExceptT parsePlutusCore))
+parse str = liftErr (runAlex str (runExceptT parsePlutusCoreProgram))
+    where liftErr (Left s)  = Left (LexErr s)
```

Comment:
might be sensible to pull `liftErr` into a top-level binding to avoid repetition?

Reply:
:+1: